suite-common/suite-utils/src/device.ts

```diff
@@ -166,7 +166,7 @@ export const shouldDisplayInitialWarningIcon = (deviceStatus: ConnectedDeviceSta
 export const isDeviceRemembered = (device?: TrezorDevice): boolean => !!device?.remember;
 
 export const getIsDeviceConnectedViaBluetooth = (device?: TrezorDevice): boolean =>
-    !!device?.bluetoothProps;
+    !!device?.bluetoothProps && device.connected;
```

Comment:
This would match better the name.
```suggestion
    !!device.connected && !!device?.bluetoothProps;
```

Reply:
Fixed